Classes/Controller/Slot/AbstractSlotController.php

```diff
@@ -44,11 +44,15 @@
 
     /**
      * @param string $key
+     * @param string $translationPrefix
      * @return string
      */
-    protected function translate($key)
+    protected function translate($key, $translationPrefix = '')
     {
-        return $GLOBALS['LANG']->sL(static::LANGUAGE_FILE .
-            ':slot.' . $this->translationPrefix . '.' . $key);
+        if (empty($translationPrefix)) {
+            $translationPrefix = $this->translationPrefix;
+        }
+
+        return $GLOBALS['LANG']->sL(static::LANGUAGE_FILE . ':slot.' . $translationPrefix . '.' . $key);
```

Comment:
This change is not necessary. Please prevent it.

Reply:
I wanted to piggyback your translation keys but success and failure have different translationPrefix values. So the translationPrefix had to be dynamic.

I see now, that new separate translation keys are probably much cleaner. I'll revert the logic.